### ISSUE TITLE
Broken: ext-componentLoader - emits 'renderer: html' client-side code

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1790]
Found: Ext.Net.7.0.0-preview4-2020-07-03
Ext.NET forum thread: [Help - Error using <ext-ComponentLoader>](https://forums.ext.net/showthread.php?62934)

A panel with a simple `<ext-componentLoader />` component triggers a client-side exception as it does not quote the [renderer config](https://docs.sencha.com/extjs/7.2.0/classic/Ext.ComponentLoader.html#cfg-renderer) (or define a corresponding variable/funcion), so in a page with this:

```xml
<ext-panel title="myPanel" width="500" height="250">
    <loader>
        <ext-componentLoader url="http://www.google.com" />
    </loader>
</ext-panel>
```

The resulting code is:
```js
Ext.create("Ext.panel.Panel", {
    height: 250,
    id: "ctl02",
    loader: {
        url: "http://www.google.com",
        renderer: html
    },
    renderTo: "App.ctl02_Container",
    width: 500,
    title: "myPanel"
});
```

Reading `renderer: html` instead of `renderer: 'html'`. In this case, it should actually suppress that line completely, as we are to assume the default (which is `'html'` because it was not specified in the component.

To overcome the issue, explicitly define the renderer as the quoted string, like this: `renderer="'html'"`:

```xml
<ext-panel title="myPanel" width="500" height="250">
    <loader>
        <ext-componentLoader url="http://www.google.com" renderer="'html'" />
    </loader>
</ext-panel>
```

**Comment by geoffreymcgill** [comment]
The goal is to have this issue fixed in time for the upcoming `preview4` release.

**Comment by AndreyChechel** [comment]
Fixed in `7.0.0-preview4`
